### PR TITLE
Feature/improve perf

### PR DESCRIPTION
**Pull request by ganfra**
Improve some perfs on timeline.
Use in memory for typing.
Fix crash on shortcuts

### CODE REVIEW

File: vector/src/main/java/im/vector/riotx/features/home/room/typing/TypingHelper.kt [hunk 1]
```diff
@@ -28,42 +27,33 @@ class TypingHelper @Inject constructor(
...
+    fun getTypingMessage(roomId: String): LiveData<String> {
+        val liveData = session.typingUsersTracker().getTypingUsersLive(roomId)
+        return Transformations.map(liveData) { typingUsers ->
+            when {
+                typingUsers.isEmpty() ->
+                    ""
+                typingUsers.size == 1 ->
+                    stringProvider.getString(R.string.room_one_user_is_typing, typingUsers[0].disambiguatedDisplayName)
+                typingUsers.size == 2 ->
+                    stringProvider.getString(R.string.room_two_users_are_typing, typingUsers[0].disambiguatedDisplayName, typingUsers[1].disambiguatedDisplayName)
```
Comment (bmarty) [inline vector/src/main/java/im/vector/riotx/features/home/room/typing/TypingHelper.kt:53]: That's better to use disambiguatedDisplayName, thanks

Comment (ganfra) [reply vector/src/main/java/im/vector/riotx/features/home/room/typing/TypingHelper.kt:53]: ??? It's already the case or I'm blinded 👀

Comment (bmarty) [reply vector/src/main/java/im/vector/riotx/features/home/room/typing/TypingHelper.kt:53]: I was just saying thank you ❤️

File: vector/src/main/java/im/vector/riotx/core/ui/list/GenericLoaderItem.kt [hunk 2]
```diff
@@ -13,7 +13,9 @@ abstract class GenericLoaderItem : VectorEpoxyModel<GenericLoaderItem.Holder>()
 
     // Maybe/Later add some style configuration, SMALL/BIG ?
 
-    override fun bind(holder: Holder) {}
+    override fun bind(holder: Holder) {
+        super.bind(holder)
+    }
```
Comment (bmarty) [inline vector/src/main/java/im/vector/riotx/core/ui/list/GenericLoaderItem.kt:18]: Remove?

**Pull request merged**